examples/lang/flipper/src/lib.rs

```diff
@@ -42,6 +52,32 @@ contract! {
     impl Flipper {
         /// Flips the current state of our smart contract.
         pub(external) fn flip(&mut self) {
+            println(&format!("flip"));
+            let mut alloc = unsafe {
+                let mut fw_alloc = storage::alloc::BumpAlloc::from_raw_parts(Key([0x0; 32]));
+                let mut dyn_alloc = storage::alloc::DynAlloc::allocate_using(&mut fw_alloc);
+                dyn_alloc.initialize(());
+                dyn_alloc
+            };
+            let mut alloc = unsafe { BumpAlloc::from_raw_parts(Key([0x0; 32])) };
+
+            for _ in 0..100 {
+                let mut inner_vec = unsafe {
+                    Vec::<i32>::allocate_using(&mut alloc).initialize_into(())
```

Comment:
This line (if it used `DynAlloc` instead of `BumpAlloc`) is what we are interested in testing >3000 times before deallocating.